torch/_inductor/lowering.py

```diff
@@ -4996,6 +4996,25 @@ def cumprod(x, axis=None, dtype=None):
     return result
 
 
+@register_lowering(aten.logcumsumexp)
+def logcumsumexp(x, dim):
+    if is_integer_dtype(x.get_dtype()) or is_boolean_dtype(x.get_dtype()):
+        dtype = torch.int64
+    else:
+        dtype = x.get_dtype()
+
+    def log_add_exp(a, b):
+        min_v = ops.minimum(a, b)
+        max_v = ops.maximum(a, b)
+        return ops.log1p(ops.exp(min_v - max_v)) + max_v
```

Comment:
We probably need this special case for infinities:
https://github.com/pytorch/pytorch/blob/1bf9ddf130a15abc7cbbe905baf138fb1477b9fe/aten/src/ATen/native/cuda/LogcumsumexpKernel.cu#L38-L44